integrations/terraform/envs/portal/config.tf

```diff
@@ -0,0 +1,124 @@
+locals {
+  # This prefix will be used for all of your created resources
+  prefix = "portal"
+  # The AWS account ID you want to add your AWS resources to
+  aws_account_id = "123456789012"
+  # Name of the AWS account
+  account_name = "data-platform"
+  # Region where you want to deploy resources to
+  aws_region = "eu-west-1"
+
+  # Choose whether you want to set up Conveyor integrations
+  conveyor_enabled = false
```

Comment:
in the default settings I currently disabled the Conveyor and Datahub installation parts. Removing them from the code base would lead to far at the moment. The most relevant parts for them are indeed definitely the environment and data product parts